controllers/application.ts

```
@@ -0,0 +1,41 @@
+import { application } from "express";
+import { ApplicationService } from "../services/applicationService";
+import { AppParams } from "../utils/types";
+
+export class Applications {
+    applicationService: ApplicationService;
+    
+    constructor() {
+        this.applicationService = new ApplicationService();
+    }
+
+    async getAllApplications() {
+        try {
+            return this.applicationService.getAllApplications();
+        } catch (err) {
+            console.log(err);
+            throw new Error("No applications was found");
+        }
+    }
+    async addApplication(appParams: AppParams) {
+        try {
+            return this.applicationService.addApplication(appParams);
+        } catch (err) {
+            console.log(err);
```

Maybe throw an Error on the same way as above? :)